Dockerfile

```diff
@@ -12,24 +12,26 @@
 # the License.
 #
 
-FROM adoptopenjdk/openjdk11:alpine
+FROM eclipse-temurin:11-jdk-alpine as builder
+WORKDIR /build
+RUN apk add --no-cache git
+COPY ./ ./
+RUN ./mvnw install -DskipTests
 
-EXPOSE 8080
-EXPOSE 8081
-EXPOSE 8443
 
-RUN addgroup -S thirdeye && \
-  adduser -S thirdeye -G thirdeye && \
-  chown thirdeye:thirdeye /home/thirdeye
+FROM eclipse-temurin:11-jre-alpine
+RUN addgroup -g 1000 thirdeye && \
```

Comment:
I'm a bit confused by this, could you 
explain the introduction of gid and sid 1000, 
and why the group is not system `-S` anymore?

Reply:
sounds good thanks!